### ISSUE TITLE
Search by other attributes

### ISSUE THREAD

**Issue by di** [chainguard-dev/rekor-search-ui#4]
Some other things that would be nice to be able to search by besides email:

- [ ] log index
- [ ] log ID
- [ ] signature
- etc

**Comment by pdeslaur** [comment]
This is mostly done. We rely on the deprecated search endpoint to power the UI. The bigger issue here would be to do our own indexing of Rekor to power this UI.

**Comment by monperrus** [comment]
yes, it would be super useful to be able to search in the "signature" field and in the "Public Key Certificate" field.
